ui/common/ContributionBar.tsx

```diff
@@ -192,7 +192,13 @@ export default function ContributionBar({
                     transition: 'transform 300ms linear',
                   }}
                 >
-                  <Image src={player.avatar} alt="" width={26} height={26} />
+                  <Image
+                    src={player.avatar}
+                    alt=""
+                    width={26}
+                    height={26}
+                    class="rounded-full"
```

Comment:
needs to use className

```suggestion
                    className="rounded-full"
```